test/unit/tsx2/input.tsx

```diff
@@ -0,0 +1,3 @@
+import * as fs from "fs";

```

Comment:
Let's rename this directory from `tsx2` to `tsx-input`.

Reply:
Will do!